app/models/plan.rb

```diff
@@ -39,7 +39,7 @@ def deletable?
   end
 
   def has_trial?
-    trial_period.present?
+    trial_period.present? && trial_period.positive?
```

Comment:
Ruby question: won't `.positive?` also check `.present?` so we can only keep one condition ?

Reply:
`nil.positive?` => `undefined method `positive?' for nil:NilClass` ;)

Reply:
Not exactly:

```ruby
nil.positive?
# => undefined method `positive?' for nil:NilClass (NoMethodError)
```

But it's related to @rsempe point

Reply:
oh yeah so we want to make sure it's present 😄 
thanks 👍